src/ameba/rule/performance/count.cr

```diff
@@ -0,0 +1,48 @@
+module Ameba::Rule::Performance
+  # This rule is used to identify usage of `size` calls that follow to object
+  # caller names `select` and `reject`.
+  #
+  # For example, this is considered invalid:
+  #
+  # ```
+  # [1, 2, 3].select { |e| e > 2 }.size
+  # [1, 2, 3].reject { |e| e < 2 }.size
+  # [1, 2, 3].select(&.< 2).size
+  # [0, 1, 2].reject(&.zero?).size
+  # ```
+  #
+  # And it should be written as this:
+  #
+  # ```
+  # [1, 2, 3].count { |e| e > 2 }
+  # [1, 2, 3].count { |e| e < 2 }
```

Comment:
this is not equal with reject, it should invert condition

Reply:
Corrected in 9fa13848bf65de8b2fae212774ebf85d7b69f7b9. Thanks @kostya